pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scikit-decide"
-version = "0.9.8.dev19+4ad63a353"
+version = "0.0.0"
```

Comment:
Why version "0.0.0"?? Do you mean "0.10.0"?

Reply:
Actually this line is updated dynamically at build (via poetry-dynamic-versioning) and should not be changed by hand. The fact that it was modified here to a specific version was a previously introduced error. But I agree it should not specially be done in this PR, it is a bit out of scope.

Reply:
I removed it

Reply:
OK I see!